docbook/gnc-locale-C.dtd

```diff
@@ -0,0 +1,82 @@
+<!-- Add master English entities that should be translated below.
+     Only parameter entities (% entities) should be translated.
+     See htps://www.w3.org/TR/REC-xml/#sec-physical-struct. -->
+
+
+
+<!-- keycaps for keycombos and shortcuts -->
+
+<!ENTITY % kc.alt '<keycap function="alt">Alt</keycap>'>
+<!ENTITY kc.alt "%kc.alt;">
+
+<!ENTITY % kc.cmd '<keycap function="command">Command</keycap>'>
+<!ENTITY kc.cmd "%kc.cmd;">
+
+<!ENTITY % kc.ctrl '<keycap function="control">Ctrl</keycap>'>
+<!ENTITY kc.ctrl "%kc.ctrl;">
+
+<!ENTITY % kc.end '<keycap function="end">End</keycap>'>
+<!ENTITY kc.end "%kc.end;">
+
+<!ENTITY % kc.enter '<keycap function="enter">Enter</keycap>'>
+<!ENTITY kc.enter "%kc.enter;">
+
+<!ENTITY % kc.esc '<keycap function="escape">Esc</keycap>'>
+<!ENTITY kc.esc "%kc.esc;">
+
+<!ENTITY % kc.home '<keycap function="home">Home</keycap>'>
+<!ENTITY kc.home "%kc.home;">
+
+<!ENTITY % kc.shift '<keycap function="shift">Shift</keycap>'>
+<!ENTITY kc.shift "%kc.shift;">
+
+<!ENTITY % kc.space '<keycap function="space">Space</keycap>'>
+<!ENTITY kc.space "%kc.space;">
+
+<!ENTITY % kc.tab '<keycap function="tab">Tab</keycap>'>
+<!ENTITY kc.tab "%kc.tab;">
+
+
+<!-- guimenus -->
+<!ENTITY % gm.file "<guimenu>File</guimenu>">
+<!ENTITY gm.file "%gm.file;">
+
+
+<!-- guimenus with accel-->
+<!ENTITY % gma.file "<guimenu><accel>F</accel>ile</guimenu>">
+<!ENTITY gma.file "%gma.file;">
+
+<!ENTITY % gma.rep "<guimenu><accel>R</accel>eports</guimenu>">
+<!ENTITY % gma.budget "<guisubmenu>B<accel>u</accel>dget</guisubmenu>">
+<!ENTITY % gma.budget-rep "<guimenuitem>Budge<accel>t</accel> Report</guimenuitem>">
+
+
+<!-- menuchoice -->
+<!ENTITY mc.rep.budget.budget-rep "<menuchoice>%gma.rep;%gma.budget;%gma.budget-rep;</menuchoice>">
+
+
+
+
+
+<!-- untranslated message -->
+<!--
+  Translators:
+    If you can only offer a partial translation, that is fine.
+    Just inform the reader and potential successors about the state by inserting
+    the "&untranslated;" entity and commenting off the still untranslated part.
+-->
+
+<!ENTITY % untranslated 'Dear Reader, we are really sorry, but nobody offered a translation
+  for the following parts. If you would like to help us, follow the instructions at <ulink
+    url="https://wiki.gnucash.org/wiki/Translation#Translating_the_GnuCash_Guide_and_Help">
+  <citetitle>Gnucash wiki: How to translate the GnuCash guide and/or help files</citetitle></ulink>.
+  Questions can be discussed at the gnucash-XX@gnucash.org mailing list.'>
+
+<!ENTITY untranslated "%untranslated;">
+

```

Comment:
I don't think wiki and Wikipedia should be moved because translated pages may not exist.
For example, if you access https://en.wikipedia.org/wiki/Canary_Islands, you can see it. However
https://de.wikipedia.org/wiki/Canary_Islands doesn't exist.
We have to use https://de.wikipedia.org/wiki/Kanarische_Inseln in Deutsch and https://ja.wikipedia.org/wiki/カナリア諸島 in 日本語.
404 error is annoying.

Reply:
I think partial URL segments should be avoided for ENTITY(s).
If you'd like to define URLs by entities, it is better to use the combination of parameter entities.

Reply:
wontfix